modules/tensor_mechanics/src/materials/ADPowerLawCreepStressUpdate.C

```diff
@@ -54,7 +54,7 @@ void
 ADPowerLawCreepStressUpdate<compute_stage>::computeStressInitialize(
     const ADReal & /*effective_trial_stress*/, const ADRankFourTensor & /*elasticity_tensor*/)
 {
-  if (_has_temp)
+  if (_temperature[_qp] != 0.0)
```

Comment:
...And added this temperature check so I could get rid of the _has_temp bool altogether. @dschwen